src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/MediaPicker/ImageListQuery/ImageListQuery.jsx

```diff
@@ -14,13 +14,15 @@ const ImageListQueryCmp = ({
     selectedPath,
     initialSelection
 }) => {
-    const {data, error, loading} = useQuery(MediaPickerImages, {
+    const {data, error, loading, refetch} = useQuery(MediaPickerImages, {
         variables: {
             typeFilter: ['jmix:image'],
             path: selectedPath
         }
     });
 
+    window.refetchCallbacks = [refetch];
```

Comment:
Is is possible to remove the added fetch entry on component unmount ? (by using `useCallback()` hook if I remember well)

Reply:
I guess, no need to remove it, as we didn't add listener for that

Reply:
I'll try something else...

Reply:
done